apps/sms/views/conversation/js/conversation.js

```diff
@@ -516,6 +527,8 @@ var ConversationView = {
   beforeEnter: function conv_beforeEnter(args) {
     this.clearConvertNoticeBanners();
     this.setHeaderAction(ActivityClient.hasPendingRequest() ? 'close' : 'back');
+    this.header.removeAttribute('no-font-fit');
```

Comment:
__nit:__ would be great to have comment here, I'm still not 100% sure what it does, should it remove these attributes for every _beforeEnter_ or only for the first time and after that it's almost noop callas